core/src/main/java/jenkins/health/HealthCheck.java

```diff
@@ -0,0 +1,25 @@
+package jenkins.health;
```

Comment:
Worth a new package when there's already `jenkins.diagnostics`? Is this sufficiently different?

Reply:
`jenkins.diagnostics` contains administrative monitors. No strong opinion here.